### ISSUE TITLE
[Feature Request] Use the same element for < and |

### ISSUE THREAD

**Issue by kud** [dwarvesf/hidden#238]
Hey,

In order to optimise the number of elements in the menubar but also having a better UX in my opinion, it could be great that `<` and `|` are the same element.

When you click on it, it expands or collapses the menu.

Thanks,

**Comment by casualbugs** [comment]
Cam in to ask for the same feature! Using two icons seems like a waste of space

**Comment by bigplayer-ai** [comment]
Yeah the always shown section is useless as you can just drag your menubar items on the right of >\<
This way we could save one icon from hiding behind the notch.
